packages/cli/src/public-api/v1/handlers/workflows/workflows.handler.ts

```diff
@@ -292,21 +309,36 @@ export = {
 
 			const workflowManager = Container.get(ActiveWorkflowManager);
 
-			if (workflow.active) {
+			if (workflow.activeVersionId !== null) {
```

Comment:
Can this ever be `undefined`? If yes, it will be considered to be active by this check. Also same comment for other `!== null` checks